spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/token/authentication/XsuaaJwtDecoderBuilder.java

```diff
@@ -46,6 +48,19 @@ public JwtDecoder build() {
 				combinedTokenValidators, postValidationActions);
 	}
 
+	/**
+	 * Assembles a ReactiveJwtDecoder
+	 *
+	 * @return ReactiveJwtDecoder
+	 */
+	public ReactiveJwtDecoder buildAsReactive() {
```

Comment:
`postValidationActions` is ignored. Even if intended, this might look like a bug to users, because they configure them in the builder, but they're not used.

Reply:
Done -> thanks

Reply:
Hi @simontb 

can you please perform a review here:
https://github.com/SAP/cloud-security-xsuaa-integration/pull/103/commits/503f9f155b6036d768d315098e5f75129fbebf80#diff-56d67c106c51045ccf1b6065f313d983R68

Thanks!!!